virt/backup_restore/virt-backup-restore-overview.adoc

```diff
@@ -6,9 +6,6 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-:FeatureName: OADP for {VirtProductName}
-include::snippets/technology-preview.adoc[]
-
 You back up and restore virtual machines by using the xref:../../backup_and_restore/index.adoc#application-backup-restore-operations-overview[OpenShift API for Data Protection (OADP)].
```

Comment:
```suggestion
Back up and restore virtual machines by using the xref:../../backup_and_restore/index.adoc#application-backup-restore-operations-overview[OpenShift API for Data Protection (OADP)].
```
For the sake of minimalism, I would remove "You" from this sentence. Perhaps out of scope of this PR; I leave it to your discretion.

Reply:
Good suggestion; thanks!